### PR TITLE
[TASK] Migrate TypoScript linting to a GitHub action

### PR DESCRIPTION
**Pull request by lukaszuznanski**
I prepared github action for typoscript linting, migrate tea extension to use it as example
https://github.com/TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint

### CODE REVIEW

**Review by oliverklee — CHANGES_REQUESTED** [review]
We also need to remove `tools/typo3-typoscript-lint` and remove it from the PHIVE configuration file.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,6 +29,16 @@ jobs:
...
+  typoscript-lint:
+    name: "TypoScript linter"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Run TypoScript lint"
+        uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+    strategy:
+      fail-fast: false
```
Comment (oliverklee) [inline .github/workflows/ci.yml:41]: This is not needed in this case it this setting does not make any difference for jobs without a matrix.

**Comment by lukaszuznanski** [comment]
> We also need to remove `tools/typo3-typoscript-lint` and remove it from the PHIVE configuration file.

Maybe we should keep it in phive and in composer.json in order to run lints locally?

**Comment by oliverklee** [comment]
> Maybe we should keep it in phive and in `composer.json` in order to run lints locally?

I hadn't thought of this … is there any way we can avoid these redundancies? Maybe execute the GitHub actions locally?

**Comment by lukaszuznanski** [comment]
> > Maybe we should keep it in phive and in `composer.json` in order to run lints locally?
> 
> I hadn't thought of this … is there any way we can avoid these redundancies? Maybe execute the GitHub actions locally?

Right now I found only "nektos/act" tool to running github actions locally, but anyway, removal of phar files and composer.json scripts will break gitlab-ci pipeline.

**Review by oliverklee — APPROVED** [review]

**Pull request merged**